bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/ItemUtil.java

```diff
@@ -39,8 +39,8 @@ private ItemUtil() {
      *
      * @return true if the specified name is a valid item name, otherwise false
      */
-    public static boolean isValidItemName(String itemName) {
-        if ((itemName != null) && (!itemName.isEmpty())) {
+    public static boolean isValidItemName(final String itemName) {
```

Comment:
Why do we need this method besides assertValidItemName? It seems to me that whereever isValidItemName is called, the caller in the end also throws an IAE.